hepcrawl/spiders/wsp_spider.py

```diff
@@ -89,16 +89,24 @@ def __init__(self, package_path=None, ftp_folder="WSP", ftp_host=None, ftp_netrc
     def start_requests(self):
         """List selected folder on remote FTP and yield new zip files."""
         if self.package_path:
-            yield Request(self.package_path, callback=self.handle_package_file)
+            dummy, new_files = local_list_files(
```

Comment:
In python, for non-used variables like this `dummy` it's 'convention' to use just `_`.